data/speakers.ts

```diff
@@ -61,19 +61,26 @@ const speakers: Speaker[] = [
   {
     objectID: 'ireaderinokun',
     fullName: 'Ire Aderinokun',
-    tags: ['Progressive Web Apps', 'CSS', 'Standards'],
+    tags: [
+      'Progressive Web Apps', 
+      'HTML', 
+      'CSS', 
+      'Javascript', 
+      'Standards', 
+      'Accessibility'
+    ],
     email: null,
-    website: 'https://ireaderinokun.com/',
+    website: 'https://ireaderinokun.com',
     social: {
       twitter: 'ireaderinokun',
       github: 'ireade',
-      blog: 'https://medium.com/@ireade',
-      linkedin: 'https://www.linkedin.com/in/iaderinokun/',
+      blog: 'https://bitsofco.de',
+      linkedin: 'https://www.linkedin.com/in/iaderinokun',
     },
     talks: [],
```

Comment:
Hi @ireade, thanks for your PR! Could you please add your talks, future, and past. If you have too many of them, just last 5 for instance :) See Nicolas Hoizey as an example, please. Thank you very much again!

Reply:
Hi, I'm merging this for now, but could you please update your talks in the new PR? Thank you.